src/main.rs

```diff
@@ -51,8 +51,13 @@ type Span = span::Span<span::ZeroIndexed>;
 pub fn main() {
     env_logger::init().unwrap();
 
-    let arg_count = ::std::env::args().count();
-    if arg_count > 1 {
+    let args: Vec<String> = ::std::env::args().collect();
```

Comment:
nit: one could do this without collecting by using `next` on the args iterator. Not really necessary though.